chroma_core/management/__init__.py

```diff
@@ -68,5 +67,3 @@ def all_subclasses(cls):
 # before our hook, so that Permission objects will be there
 # by the time we are called.
 import django.contrib.auth.management
-
-post_migrate.connect(setup_groups)
```

Comment:
How does `setup_groups` get called now?

Reply:
It gets called here:

https://github.com/whamcloud/integrated-manager-for-lustre/pull/1202/files?file-filters%5B%5D=#diff-ea98e1400611bc4bbb2e0d030794f7d1R18

post migration hooks are now only safe to call within the ready block of an `AppConfig`